media-sound/mpg321/mpg321-0.3.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A simple and lightweight command line MP3 Player"
+DESCRIPTION="mpg321 is a very popular command-line mp3 player. mpg321 is used \
+for frontends, as an mp3 player and as an mp3 to wave file decoder (primarily \
+for use with CD-recording software)."
+HOMEPAGE="http://mpg321.sourceforge.net/"
+COPYRIGHT="2001 Joe Drew
+		2006-2012 Nanakos Chrysostomos"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ncu.dl.sourceforge.net/project/mpg321/mpg321/$portVersion/mpg321_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="056fcc03e3f5c5021ec74bb5053d32c4a3b89b4086478dcf81adae650eac284e"
+SOURCE_DIR="mpg321-$portVersion-orig"
+PATCHES="mpg321-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86 x86_gcc2"
+
+PROVIDES="
+	mpg321$secondaryArchSuffix = $portVersion
+	cmd:mpg321$secondaryArchSuffix = $portVersion
+	cmd:mpg123$secondaryArchSuffix = $portVersion
```

Comment:
I would remove the **`cmd:mpg123$secondaryArchSuffix = $portVersion`** because it is provided by another package which we already have.
But if you were asked, by another contributor, to add this line then I guess this is done on purpose.